examples/file-sharing.rs

```diff
@@ -69,9 +68,9 @@
 //!
 //!    ```
 //!    cargo run --example file-sharing -- \
-//!              --peer /ip4/127.0.0.1/tcp/40837/p2p/12D3KooWPjceQrSwdWXPyLLeABRXmuqt69Rg3sBYbU1Nft9HyQ6X \
-//!              get \
-//!              --name <name-for-others-to-find-your-file>
+//!              --peer
+//! /ip4/127.0.0.1/tcp/40837/p2p/12D3KooWPjceQrSwdWXPyLLeABRXmuqt69Rg3sBYbU1Nft9HyQ6X \             
```

Comment:
I am surprised that rustfmt formats code snippets. Is that the default?

Reply:
Uhm, I think in this case it is not about code snippets but just comments in general.